tests/ParserTest.elm

```diff
@@ -551,6 +551,36 @@ expr =
                   , "'A'"
                   , Ok (Literal (Char 'A'))
                   )
+                  -- https://github.com/elm/compiler/blob/dcbe51fa22879f83b5d94642e117440cb5249bb1/compiler/src/Parse/String.hs#L279-L285
+                , ( "escape n"
+                  , "'\\n'"
+                  , Ok (Literal (Char '\n'))
+                  )
+                , ( "escape r"
+                  , "'\\r'"
+                  , Ok (Literal (Char '\r'))
+                  )
+                , ( "escape t"
+                  , "'\\t'"
+                  , Ok (Literal (Char '\t'))
+                  )
+                , ( "double quote"
+                  , "'\\\"'"
+                  , Ok (Literal (Char '"')) -- "
+                  )                         -- ^ workaround for official elm
+                                            --   vscode syntax highlighter
```

Comment:
:see_no_evil: OK

Reply:
If you want, I'm happy to remove it.
Without it, however, the coloring is wrong for the next ~17 lines and a "parser error" prevents vscode from showing more helpful errors :-/

https://github.com/Krzysztof-Cieslak/vscode-elm/issues/244